js/src/forum/components/Search.js

```diff
@@ -200,7 +202,8 @@ export default class Search extends Component {
     window.addEventListener('resize', this.updateMaxHeightHandler);
   }
 
-  onremove() {
+  onremove(vnode) {
+    super.onremove(vnode);
     window.removeEventListener('resize', this.updateMaxHeightHandler);
```

Comment:
Just to match the rest of the changes

```suggestion
    super.onremove(vnode);

    window.removeEventListener('resize', this.updateMaxHeightHandler);
```